src/pages/transactions/[txid].page.tsx

```diff
@@ -152,6 +182,35 @@ function getDfTx(vouts: TransactionVout[]): DfTx<any> | undefined {
   return (stack[1] as OP_DEFI_TX).tx;
 }
 
+function getNetworkParams(network: EnvironmentNetwork): string {
```

Comment:
lets avoid duplication, we already have same function in `src/pages/transactions/[txid].page.tsx`, can we create a common file and place this function to avoid duplication?